tests/unit/test_charm.py

```diff
@@ -138,3 +148,61 @@ def simulate_lifecycle(harness):
     )()
     event = type("Event", (), {"app": app, "relation": relation})()
     harness.charm._on_admin_relation_changed(event)
+
+
+class TestState(TestCase):
```

Comment:
are these tests the same on each charm that uses state?

Reply:
Yes